src/Command/MakerCommand.php

```diff
@@ -79,7 +82,7 @@ protected function interact(InputInterface $input, OutputInterface $output)
                 continue;
             }
 
-            $value = $this->io->ask($argument->getDescription(), $argument->getDefault(), [Validator::class, 'notBlank']);
+            $value = $this->io->ask($argument->getDescription(), $argument->getDefault(), array(Validator::class, 'notBlank'));
```

Comment:
Please, revert this change. We don't use the legacy `array()` notation in this bundle.

Reply:
And it's available since PHP 5.4. Anyone using this bundle will probably have at least 5.5 or 5.6